assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -306,13 +306,19 @@ export default {
 
         // Overrides
 
+        // @computed()
```

Comment:
@merefield Please remove this commented code

Reply:
Done, apologies